.env.template

```diff
@@ -172,3 +172,12 @@ TW_CONSUMER_KEY=
 TW_CONSUMER_SECRET=
 TW_ACCESS_TOKEN=
 TW_ACCESS_TOKEN_SECRET=
+
+################################################################################
+### EMAIL (SMTP)
+################################################################################
+
```

Comment:
Add an USE_EMAIL variable that can be set to True or False
`USE_EMAIL=False`
and initialize `self.use_email' in config.py